hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/contract/ContractTestUtils.java

```diff
@@ -1102,7 +1101,15 @@ public static void validateFileContent(byte[] concat, byte[][] bytes) {
                 mismatch);
   }
 
-  public static void validateVectoredReadResult(List<FileRange> fileRanges, byte[] DATASET)
+  /**
+   * Utility to validate vectored read results.
+   * @param fileRanges input ranges.
+   * @param originalData original data.
+   * @throws ExecutionException any exception.
```

Comment:
the future io stuff will unwrap execution exceptions. so remove the catch/fail there, just let it all get passed up to the test case